templates/web-application/.stylelintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://json.schemastore.org/stylelintrc",
+    "extends": "@workleap/stylelint-config"
+}
```

Comment:
New file for [@workleap/stylelint-config](https://github.com/workleap/wl-web-configs/tree/main/packages/stylelint-config)